humanguide/other-aspects/01-convergence.rst

```diff
@@ -0,0 +1,19 @@
+Convergence
+===========
+
+Convergence is one of the milestones Ubuntu Touch wants to achieve. One
+app, usable on all platforms: mobile, tablet, desktop.
```

Comment:
Usually I define convergence as "available across all screen sizes and input methods, *such as* mobile handsets, tablets, and laptops."

It may therefore be useful to have two sub-headings on this page, one for screen sizes and the other for input methods.